<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaRoutePlanKSK</class>
 <widget class="QWidget" name="StaRoutePlanKSK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StaRoutePlanKSK</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,5">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QGroupBox" name="trainRouteBox">
       <property name="title">
        <string>列车进路序列 只读(不可修改)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>调车进路 只读(不可修改)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QGroupBox" name="shuntRouteBox">
            <property name="title">
             <string>调车进路序列</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="hookPlanBox">
            <property name="title">
             <string>勾序列</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>作业单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>单头信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,5">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>车站</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stationCombo">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="limitsButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>释放权限</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>字体</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSizeCombo">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>常规</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>大</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>超大</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>极大</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,1">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>56</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>56</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>历史进路</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>状态:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
